diff --git a/uct/uct.c b/uct/uct.c
index 4d0a299..79fc35d 100644
--- a/uct/uct.c
+++ b/uct/uct.c
@@ -750,7 +750,7 @@ uct_state_init(char *arg, struct board *b)
 	u->dumpthres = 0.01;
 	u->playout_amaf = true;
 	u->amaf_prior = false;
-	u->max_tree_size = 1408ULL * 1048576;
+	u->max_tree_size = 500 * 1048576;
 	u->fast_alloc = true;
 	u->pruning_threshold = 0;
 
@@ -1037,7 +1037,7 @@ uct_state_init(char *arg, struct board *b)
 			} else if (!strcasecmp(optname, "max_tree_size") && optval) {
 				/* Maximum amount of memory [MiB] consumed by the move tree.
 				 * For fast_alloc it includes the temp tree used for pruning.
-				 * Default is 3072 (3 GiB). */
+				 * Default is 500 (500 MiB). */
 				u->max_tree_size = atol(optval) * 1048576;
 			} else if (!strcasecmp(optname, "fast_alloc")) {
 				u->fast_alloc = !optval || atoi(optval);
